### PR TITLE
[office-js-preview] Updating office-js-preview with beta CDN content

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -18497,7 +18497,7 @@ declare namespace Excel {
         changeType: Excel.DataChangeType | "Unknown" | "RangeEdited" | "RowInserted" | "RowDeleted" | "ColumnInserted" | "ColumnDeleted" | "CellInserted" | "CellDeleted";
         /**
          *
-         * Represents the information about the change detail
+         * Represents the information about the change detail. This property can be retrieved when the Changed event is triggered on single cell. If the Changed event is triggered on multiple cells, this property can not be retrieved.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Represents the information about the change detail. This property can be retrieved when the Changed event is triggered on a single cell. If the Changed event is triggered on multiple cells, this property cannot be retrieved.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -18588,6 +18588,55 @@ declare namespace Excel {
          */
         getRangeOrNullObject(ctx: Excel.RequestContext): Excel.Range;
     }
+    /**
+     *
+     * Provides information about the worksheet format change event.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Seems like this should be something like:
```suggestion
     * Provides information about the worksheet row hidden change event.
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -18588,6 +18588,55 @@ declare namespace Excel {
...
+         *
+         * Gets the range address that represents the changed area of a specific worksheet.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        address: string;
+        /**
+         *
+         * Gets the change type that represents how the Changed event is triggered. See Excel.DataChangeType for details.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Seems like this should be:
```suggestion
         * Gets the change type that represents how the Changed event is triggered. See Excel.RowHiddenChangeType for details.
```

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -18639,7 +18688,7 @@ declare namespace Excel {
         worksheetId: string;
         /**
          *
-         * Represents the information about the change detail
+         * Represents the information about the change detail. This property can be retrieved when the Changed event is triggered on single cell. If the Changed event is triggered on multiple cells, this property can not be retrieved.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Represents the information about the change detail. This property can be retrieved when the Changed event is triggered on a single cell. If the Changed event is triggered on multiple cells, this property cannot be retrieved.
```

File: types/office-js-preview/index.d.ts [hunk 5]
```diff
@@ -21185,6 +21264,16 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.1]
          */
         readonly rowIndex: number;
+        /**
+         *
+         * Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: This seems incomplete.

Also, won't this all end up being on one line in the docs?

File: types/office-js-preview/index.d.ts [hunk 6]
```diff
@@ -36280,17 +36360,17 @@ declare namespace Excel {
         getItemAt(index: number): Excel.Comment;
         /**
          *
-         * Gets the comment from the specified cell.
+         * Gets the comment from the specific cell.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: To me this should remain "specified"

File: types/office-js-preview/index.d.ts [hunk 7]
```diff
@@ -36280,17 +36360,17 @@ declare namespace Excel {
...
          * @beta
          *
-         * @param cellAddress The cell which the comment is on. This can be a Range object or a string. If it's a string, it must contain the full address, including the sheet name.
+         * @param cellAddress The cell which the comment is on. This can be a Range object or a string. If it's a string, it must contain the full address, including the sheet name. An `InvalidArgument` error is thrown if the provided range is larger than one cell.
          */
         getItemByCell(cellAddress: Range | string): Excel.Comment;
         /**
          *
-         * Gets a comment from the collection with the corresponding reply ID.
+         * Gets a comment related to its reply ID in the collection.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Gets a comment by its reply ID from the collection.
```

File: types/office-js-preview/index.d.ts [hunk 8]
```diff
@@ -40109,6 +40189,24 @@ declare namespace Excel {
...
         cellDeleted = "CellDeleted"
     }
+    /**
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    enum RowHiddenChangeType {
+        /**
+         *
+         * Unhidden represents the hidden state of row is Unhidden.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Unhidden represents the row is not hidden.
```

File: types/office-js-preview/index.d.ts [hunk 9]
```diff
@@ -40109,6 +40189,24 @@ declare namespace Excel {
...
+         *
+         */
+        unhidden = "Unhidden",
+        /**
+         *
+         * Hidden represents the hidden state of row is hidden.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Hidden represents the row is hidden.
```

File: types/office-js-preview/index.d.ts [hunk 10]
```diff
@@ -40281,7 +40379,13 @@ declare namespace Excel {
          * WorksheetSingleClicked represents the type of event registered on worksheet, and occurs when cell is left-clicked/tapped.
          *
          */
-        worksheetSingleClicked = "WorksheetSingleClicked"
+        worksheetSingleClicked = "WorksheetSingleClicked",
+        /**
+         *
+         * WorksheetRowHiddenChanged represents the type of event registered on worksheet, and occurs when there is a row hidden changed.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * WorksheetRowHiddenChanged represents the type of event registered on worksheet, and occurs when a row's hidden state is changed.
```

File: types/office-js-preview/index.d.ts [hunk 11]
```diff
@@ -50345,6 +50449,16 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.1]
              */
             rowIndex?: number;
+            /**
+             *
+             * Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: This seems incomplete.

Also, won't this all end up being on one line in the docs?

File: types/office-js-preview/index.d.ts [hunk 12]
```diff
@@ -56423,6 +56537,16 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.1]
              */
             rowIndex?: boolean;
+            /**
+             *
+             * Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: This seems incomplete.

Also, won't this all end up being on one line in the docs?

File: types/office-js-preview/index.d.ts [hunk 13]
```diff
@@ -63740,6 +63864,16 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.1]
              */
             rowIndex?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: This seems incomplete.

Also, won't this all end up being on one line in the docs?

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Approving with the understanding that if you opt to take any of these changes they may need to be pushed upstream.

**Comment by RyanCavanaugh** [comment]
@AlexJerabek should I merge this or do you want to address the above comments first?

**Comment by AlexJerabek** [comment]
@RyanCavanaugh Updated and ready to go. Thanks!

File: types/office-js-preview/index.d.ts [hunk 14]
```diff
@@ -21185,6 +21264,16 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.1]
          */
         readonly rowIndex: number;
+        /**
+         *
+         * Represents if ALL the cells would be saved as an array formula
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: If this all gets wrapped together, then I recommend a period at the end of this sentence.

```suggestion
         * Represents if ALL the cells would be saved as an array formula.
```

Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: ditto for later occurrences.

File: types/office-js-preview/index.d.ts (hunk 9 above)
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: Copy-paste? Seems like this should be:
```suggestion
         * Hidden indicates the row is hidden.
```

File: types/office-js-preview/index.d.ts (hunk 5 above)
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: Sorry, but this statement is still not making sense to me...

**Review by ElizabethSamuel-MSFT — CHANGES_REQUESTED** [review]
Just a couple of things to clarify.

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT.  Updated!

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**